### DISCUSSION TITLE
New task symfony translator

### DISCUSSION THREAD

**Discussion by talhaamir4130** [phpro/grumphp#939] [Ideas]
| Q               | A
| --------------- | ---
| Version         | `vendor/bin/grumphp -V`
| Bug?            | no
| New feature?    | yes
| Question?       | yes
| Documentation?  | yes
| Related tickets | comma-separated list of related tickets

Can we add support for symfony translator ?

**Comment by veewee** [comment]
Moved this issue to a discussion instead; Can you elaborate on how this task should work, what tools it executes, ... ?

**Reply by talhaamir4130** [reply]
It helps you find missing or unused translations in your App.

You can find documentation [here](https://symfony.com/doc/current/translation/debug.html)

**Reply by veewee** [reply]
You could already do this by using the shell task.

It might be a better idea to add a symfony console task in which you can define what console command you want to run. That might be a bit more flexible: it could allow you to run any symfony console command before committing your code e.g. 

Feel free to play around with the idea in a PR.
